racker/salus-telemetry-monitor-management
# Added a summary field to DetailedMonitorOutput

# Resolves

https://jira.rax.io/browse/SALUS-746

# What

Web frontends for Salus will need a way to convey the difference between several monitors all with the same type.

# How

Introduces a marker-annotation that is picked up by `MonitorConversionService` (not to be confused with "monitor translation") when building `DetailedMonitorOutput` objects returned from the monitor REST API.

## How to test

Updated existing tests to verify the annotated fields specific to each plugin type.
